src/presentation/pages/OngRegister.tsx

```diff
@@ -0,0 +1,137 @@
+                            />
+                        </div>
+                    </article>
+                </section>
+                <Divider className="my-6" />
+                <section className="flex flex-col gap-2">
+                    <h3 className="text-xl font-bold">Contato:</h3>
+                    <Input
+                        placeholder="Telefone (00) 0 0000-0000"
+                        variant="bordered"
+                        size="lg"
+                    />
+                </section>
+                <Divider className="my-6" />
+                <section className="flex flex-col gap-2">
+                    <h3 className="text-xl font-bold">
+                        Ajude-nos a conhecer a ONG:
+                    </h3>
+                    <article className="flex flex-col gap-6">
+                        <Textarea
+                            placeholder="Descreva seus programas e atividades"
+                            minRows={4}
+                            maxRows={8}
+                        />
+                        <Textarea
+                            placeholder="Missão da ONG"
+                            minRows={4}
+                            maxRows={8}
+                        />
+                        {/* FIXME: Sobrepor placeholder */}
+                        <Input
+                            type="date"
+                            max={new Date().toISOString().split('T')[0]}
+                            min="1900-01-01"
+                            placeholder="Data de fundação"
+                            variant="bordered"
+                            size="lg"
+                        />
+                    </article>
+                </section>
+                <Divider className="my-6" />
+                <section className="flex flex-col gap-2">
+                    <h3 className="text-xl font-bold">Dados de acesso:</h3>
+                    <article className="flex flex-col gap-6">
+                        <Input
+                            placeholder="Senha"
+                            variant="bordered"
+                            size="lg"
+                        />
+                        <Input
+                            placeholder="Confirmar senha"
+                            variant="bordered"
+                            size="lg"
+                        />
+                    </article>
+                </section>
+                <Divider className="my-6" />
+                <section className="flex flex-col gap-6">
+                    <Button color="primary" variant="solid" size="md">
+                        Finalizar Cadastro
+                    </Button>
+                    <p className="justify-center text-lg font-light flex gap-2">
+                        Já possui conta?
+                        <Link
+                            // TODO: Adicionar redirecionamentos
+                            href={''}
```

Comment:
por enquanto deixar assim
```suggestion
                            href={"#"}
```